src/SonarQube.Client.Tests/SonarQubeService_GetTaintVulnerabilitiesRequest.cs

```diff
@@ -54,7 +54,7 @@ public async Task GetTaintVulnerabilitiesAsync_Response_From_SonarQube()
         {
             await ConnectToSonarQube("8.6.0.0");
 
-            SetupRequest("api/issues/search?projects=shared&statuses=OPEN%2CCONFIRMED%2CREOPENED&types=VULNERABILITY&p=1&ps=500", @"
+            SetupRequest("api/issues/search?projects=shared&statuses=OPEN%2CCONFIRMED%2CREOPENED%2CRESOLVED&types=VULNERABILITY&p=1&ps=500", @"
```

Comment:
I tried passing no statuses at all, and bizarrely I then get back more issues with corrupted info that I cannot fetch the source code for them. This is what we use to fetch suppressions normally so I added it, and that works.